chore: temporarily remove search filters and update responsive UI styling

ArturBabii opened the pull request:
Temporarily removed the search filtering block from the main page to reduce distractions during development.
Improved UI styles for featured tags, adjusted spacing and font sizes, and fixed responsive behavior across devices.
![image](https://github.com/user-attachments/assets/ac17e70f-5c50-4157-a829-165404319f9b)
![image](https://github.com/user-attachments/assets/5be148d4-a6e2-460f-a911-991aa760210f)
![image](https://github.com/user-attachments/assets/4c5bc1a5-a4de-4bac-bd1c-05d67727e322)

AlexSaveliev1 commented:
![CleanShot 2025-04-17 at 11 04 56](https://github.com/user-attachments/assets/00697bcd-4184-447f-b145-165d46ca8f87)

No `cursor: pointer` when hovering on the car card

AlexSaveliev1 commented on style/style.css:
```diff
@@ -342,4 +345,123 @@ body {
\ No newline at end of file
+}
+
+/* Для планшетів */
```
Use English-only in code

ArturBabii replied:
Fixed!

AlexSaveliev1 commented on style/style.css:
```diff
@@ -342,4 +345,123 @@ body {
+    }
+}
+
+/* Для телефонів */
```
English-only

ArturBabii replied:
Fixed!

AlexSaveliev1 commented:
![CleanShot 2025-04-17 at 11 08 28](https://github.com/user-attachments/assets/36d6bbcd-ec95-4fb2-a137-9ab53bc8b97b)

1. Перенеси Featured filters для планшетів і мобільних щоб він був вертикально над фільтрами
2. Карточки розтягни на більшу ширину, лиши відступи по боках сторінки

AlexSaveliev1 commented on about.html:
```diff
@@ -19,21 +19,18 @@ <h1>About Us</h1>
-   </footer>
-  
+  </footer>
+
```
Тут відступ не потрібен

ArturBabii replied:
Fixed!

AlexSaveliev1 commented on index.html:
```diff
@@ -30,7 +30,7 @@ <h1>Car Sales</h1>
 
   <div class="container">
-    <div class="sidebar">
+    <!-- <div class="sidebar">
```
Видали закоментований код, наразі він буде тобі тільки заважати
Коли потрібно буде повернути фільтри для цього ти використаєш коміт-історію

AlexSaveliev1 commented:
Зроби відступи в CSS як і в HTML, 2 пробіли. Попитай у ChatGPT як це можна налаштувати у VSCode

ArturBabii commented:
Updated styles, fixed the placement in the item, as well as the placement of the content in the item item 
![image](https://github.com/user-attachments/assets/d6ca3811-f2aa-45e6-bc95-ecf38c8e2475)
![image](https://github.com/user-attachments/assets/62e34b48-2aba-407f-98c1-915cdec1a621)

AlexSaveliev1 reviewed (approved):
Good job!
